src/main/resources/verify-service-provider-env.yml

```diff
@@ -1,54 +0,0 @@
-# This configuration will be used when no YAML file is specified.
```

Comment:
These are all the same environment variables mentioned in configuration/verify-service-provider.yml